README copy.md

```diff
@@ -0,0 +1,58 @@
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Todo List
+
+> To-do list" is a tool that helps to organize your day. It simply lists the things that you need to do and allows you to mark them as complete.
+
+![screenshot](screenshot/view.png)
+
+## Built With
+
+- JavaScript, CSS, HTML
+- Webpack
+
+## Live Demo
+
+Not available
+
```

Comment:
- I think it would be better if you don't have a Live Demo, to delete that section on your README and add it again when need it.